Resources/config/yui.yml

```diff
@@ -31,6 +31,7 @@ system:
                         - 'ez-locationviewview'
                         - 'ez-locationviewviewservice'
                         - 'ez-serversideview'
+                        - 'ez-contenttypeserversideview'
```

Comment:
actually you should introduce the ContentTypeEditServerSideView as what you are developing is completely specific to the content type edition